### ISSUE TITLE
ES6 "TypeError: Cannot read property 'length' of undefined"

### ISSUE THREAD

**Issue by andreasgrimm** [olov/ng-annotate#128]
I tried to annotate the following function:

```javascript
export default function MyController ( log, myService ) {
  'ngInject';

  var vm = this;

  vm.data = myService.data;
  /////////////////////

  log.debug( 'myController was instantiated' );
}
```

and got the following error:
```
ng-annotate -ar --es6 ./myController.js

/usr/lib/node_modules/ng-annotate/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:29
            for (var idx = 0; idx < nodeProps.length; idx++) {
                                             ^
TypeError: Cannot read property 'length' of undefined
    at visit (/usr/lib/node_modules/ng-annotate/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:29:46)
    at visit (/usr/lib/node_modules/ng-annotate/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:41:25)
    at traverse (/usr/lib/node_modules/ng-annotate/node_modules/ordered-ast-traverse/ordered-ast-traverse.js:54:5)
    at new Lut (/usr/lib/node_modules/ng-annotate/build/es5/lut.js:22:5)
    at ngAnnotate (/usr/lib/node_modules/ng-annotate/build/es5/ng-annotate-main.js:1017:15)
    at runAnnotate (/usr/lib/node_modules/ng-annotate/build/es5/ng-annotate.js:171:15)
    at fs.js:272:14
    at Object.oncomplete (fs.js:108:15)
```

removing the es6 specific `export default` makes it work just fine.

**Comment by olov** [comment]
ng-annotate still doesn't grok ES6 code very well, see #64 and https://github.com/olov/ng-annotate/issues/64#issuecomment-61659079 in particular. Can you run ng-annotate on the output from Traceur/6to5?

**Comment by andreasgrimm** [comment]
yes. traceur output works just fine.

**Comment by olov** [comment]
Fixed crash in 0.15.2, but please continue running ng-annotate on ES5 input because it does not yet understand ES6.
